File: app/src/main/java/dev/ragnarok/fenrir/mvp/view/ISimpleOwnersView.java
package dev.ragnarok.fenrir.mvp.view;

import java.util.List;

import dev.ragnarok.fenrir.model.Owner;
import dev.ragnarok.fenrir.mvp.core.IMvpView;
import dev.ragnarok.fenrir.mvp.view.base.IAccountDependencyView;


public interface ISimpleOwnersView extends IMvpView, IErrorView, IAccountDependencyView {
    void displayOwnerList(List<Owner> owners);

    void notifyDataSetChanged();

    void notifyDataAdded(int position, int count);

    void displayRefreshing(boolean refreshing);

    void showOwnerWall(int accountId, Owner owner);
}

File: app/src/main/java/dev/ragnarok/fenrir/mvp/view/ISimpleOwnersView.kt
package dev.ragnarok.fenrir.mvp.view

import dev.ragnarok.fenrir.model.Owner
import dev.ragnarok.fenrir.mvp.core.IMvpView
import dev.ragnarok.fenrir.mvp.view.base.IAccountDependencyView

interface ISimpleOwnersView : IMvpView, IErrorView, IAccountDependencyView {
    fun displayOwnerList(owners: List<Owner>)
    fun notifyDataSetChanged()
    fun notifyDataAdded(position: Int, count: Int)
    fun displayRefreshing(refreshing: Boolean)
    fun showOwnerWall(accountId: Int, owner: Owner)
}